scripts/utils.sh

```diff
@@ -147,3 +147,22 @@ function get_ipv4_address {
     echo "${hipv4}"
     return 0
 }
+
+function verify_nvidia-smi {
+    if command_exists "nvidia-smi"; then
+        nvidia-smi --version
+        ec=$?
+        if [ ${ec} -eq 0 ]; then
+            echo "Command 'nvidia-smi' found. Installing NVIDIA drivers for use in prefix shell..."
```

Comment:
echo is no longer accurate
```suggestion
            echo "Command 'nvidia-smi' found..."
```